### PR TITLE
Asm 11497 release steampipe plugin akeyless

### PR DESCRIPTION
**Pull request by michelsk**
release steampipe plugin akeyless ppipeline

### CODE REVIEW

File: .goreleaser.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+version: 2
```
Comment (OriBenHur-akeyless) [inline .goreleaser.yml:1]: Why this file is on the root path and hidden?

Comment (michelsk) [reply .goreleaser.yml:1]: this is the convention for goreleaser:
https://goreleaser.com/how-it-works/

**Review by OriBenHur-akeyless — APPROVED** [review]

File: .github/workflows/merge_main.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    branches:
+      - main
+
+jobs:
+  release:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
```
Comment (niv-ez) [inline .github/workflows/merge_main.yaml]: remove

File: .github/workflows/merge_main.yaml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+on:
+  workflow_dispatch:
+  deployment:
+  push:
+    branches:
+      - main
+
+jobs:
+  release:
+    runs-on: ubuntu-20.04
```
Comment (niv-ez) [inline .github/workflows/merge_main.yaml]: ubuntu-22.04

File: .github/workflows/release_steampipe.yaml [hunk 4]
```diff
@@ -1,8 +1,89 @@
...
+        exit 1
+      fi
+
+   - name: Run GoReleaser
+     uses: goreleaser/goreleaser-action@v6
+     with:
+      version: '~> v2'
+      args: release --clean --skip=publish --timeout=60m
+
+   - name: List Build Artifacts
```
Comment (niv-ez) [inline .github/workflows/release_steampipe.yaml:66]: remove

**Review by niv-ez — APPROVED** [review]

File: .github/workflows/merge_main.yaml (hunk 2 above)
Comment (michelsk) [reply .github/workflows/merge_main.yaml]: done

File: .github/workflows/merge_main.yaml (hunk 3 above)
Comment (michelsk) [reply .github/workflows/merge_main.yaml]: done

File: .github/workflows/release_steampipe.yaml (hunk 4 above)
Comment (michelsk) [reply .github/workflows/release_steampipe.yaml:66]: done

**Pull request merged**